client/components/theme/style.scss

```diff
@@ -85,19 +86,19 @@ $theme-info-height: 54px;
 
 .theme__badge-price {
 	flex: 0 0 auto;
-	padding: 18px 10px 0;
+	padding: 0 10px;
 	color: var( --color-success );
 	font-size: $font-body-small;
 	font-weight: 600;
 }
 
-.theme__badge-price-test {
-	padding: 18px 5px;
-}
+// .theme__badge-price-upsell {
+// 	padding: 18px 5px;
+// }
```

Comment:
I don't understand what this is doing yet, so I would like to do that before removing it.